airbyte-integrations/connectors/source-google-pagespeed-insights/source_google_pagespeed_insights/spec.yaml

```diff
@@ -0,0 +1,26 @@
+documentationUrl: https://docs.airbyte.io/integrations/sources/google-pagespeed-insights
+connectionSpecification:
+  $schema: http://json-schema.org/draft-07/schema#
+  title: Google PageSpeed Insights Spec
+  type: object
+  required:
+    - api_key
```

Comment:
From reading the docs, it looks like having an api key is optional, but that using the API without it is heavily throttled.

Maybe we should make this property optional?

Reply:
Good idea, thanks for that suggestion. I'll add it as soon as I can.